packages/node_modules/@ciscospark/widget-base/src/browser-globals.js

```diff
@@ -88,7 +88,7 @@ function setupWidgets() {
           // Otherwise create new widget store
           return createNewWidget(widgetEl);
         }
-        console.warning(`WARNING: ciscospark.widget: No HTML element provided.`); // eslint-disable-line no-console
+        console.warn(`WARNING: ciscospark.widget: No HTML element provided.`); // eslint-disable-line no-console
```

Comment:
I didn't think so since if you use jquery with invalid selectors it would just return undefined.